src/odata-to-abstract-sql.ts

```diff
@@ -242,18 +242,18 @@ export class OData2AbstractSQL {
 	private checkAlias: (alias: string) => string;
 
 	constructor(private clientModel: AbstractSqlModel) {
-		const MAX_ALIAS_LENGTH = 64;
+		const MAX_ALIAS_LENGTH = 63;
```

Comment:
This is important for the
```ts
return randStr + alias.slice(randStr.length + alias.length - MAX_ALIAS_LENGTH)
```
below, and the switch to `<=` elsewhere was just to keep that logic consistent.  It would technically be possible to use
```ts
return randStr + alias.slice(1 + randStr.length + alias.length - MAX_ALIAS_LENGTH)
```
or similar instead, but I thought this was nicer/clearer